plugins/ibm_qradar/icon_ibm_qradar/util/utils.py

```diff
@@ -0,0 +1,35 @@
+"""Include util methods."""
+import json
+from insightconnect_plugin_runtime.exceptions import (
+    ConnectionTestException,
+    PluginException,
+)
+
+from icon_ibm_qradar.util.constants.constant import SUCCESS_RESPONSE_CODE
+
+
+def get_default_header():
+    """To get the default headers."""
+    return {"Accept": "application/json"}
+
+
+def handle_response(response):
+    """
+    To handles the http response.
+
+    :param response:
+    :return:
+    """
+    try:
+        response_data = response.json()
+    except json.decoder.JSONDecodeError as err:
+        raise PluginException(preset=PluginException.Preset.INVALID_JSON, data=err)
+
+    if response.status_code in SUCCESS_RESPONSE_CODE:
+        return {"data": response_data}
```

Comment:
```suggestion
        return response_data
```
Adjusting this to fit the comments I left in the action.py files